src/components/Properties/PropertyMultipleText.vue

```diff
@@ -192,4 +192,14 @@ export default {
 		}
 	}
 }
+::v-deep .v-select.select {
+	min-width: 0 !important;
+	flex: 1 auto;
+}
+// Mobile tweaks
+@media (max-width: 600px) {
+	.property__value {
+		width: 125px;
+	}
+}
```

Comment:
This piece of code is duplicated. Could you please put it into `css/Properties/Properties.scss`?

Reply:
thats a very good point, BUT, adding it to Properties.scss breaks the _Address book_ and _Contact groups_ on edit mode, because (im assuming) fields like this one have only the right part editable and our main problem with fields its a space issue.

Reply:
But the override already is in ContactDetails.vue, right? Doesn't that mean that it is global (for the whole form) anyway?

I'll test again.

Reply:
I deduplicated the select styling.